tests/compile-fail/no_starting_state.stderr

```diff
@@ -1,10 +1,10 @@
-error: No starting state defined, indicate the starting state with a *.
- --> $DIR/no_starting_state.rs:5:1
-  |
-5 | / statemachine! { //~ ERROR No starting state defined, indicate the starting state with a *
-6 | |     State1 + Event1 = State2,
-7 | |     State2 + Event2 = State3,
-8 | | }
-  | |_^
-  |
-  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+error: expected `:`
```

Comment:
This error seems wrong to me, compared to before

Reply:
Ohh! Good catch. I will take a look at it